packages/tools/webpack-fluid-loader/src/loader.ts

```diff
@@ -373,18 +373,26 @@ export async function start(
 }
 
 async function getFluidObjectAndRender(container: IContainer, url: string, div: HTMLDivElement) {
-	const response = await container.request({
-		headers: {
-			mountableView: true,
-		},
-		url,
-	});
+	const entryPoint = await container.getEntryPoint?.();
```

Comment:
Webpack loader now gets the component to render through the Container's entryPoint instead of issuing a special request to it.